### PR TITLE
add button component and minor adjustments

### PR DESCRIPTION
**Pull request by Matylda-Lewandowska**
minor adjustments and buttons as components

### CODE REVIEW

File: src/components/utils/ButtonFavs/buttonFavs.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        border-color: $color-primary-lm;
+      }
+  
+      &::before {
+        content: $icon-favs-full;
+        color: $color-red-lm; 
+      }
+    }
+  }
+
```
Comment (Meg-Sowka) [inline src/components/utils/ButtonFavs/buttonFavs.scss]: please remove empty lines

File: src/components/utils/ButtonFavs/buttonFavs.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
+@import '../../../styles/utils/variables';
+
+
```
Comment (Meg-Sowka) [inline src/components/utils/ButtonFavs/buttonFavs.scss]: please remove empty lines

**Review by Meg-Sowka — CHANGES_REQUESTED** [review]

File: src/components/utils/ButtonDefault/buttonDefault.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  align-items: center;
+  text-align: center;
+  flex: 1 0 0;
+  box-shadow: 0 0 0 0 $color-primary-lm;
+  
+  transition: box-shadow $animation-duration;
+  
+  &:hover {
+    box-shadow: 0px 3px 13px 0px $color-primary-lm;
+
```
Comment (Meg-Sowka) [inline src/components/utils/ButtonDefault/buttonDefault.scss]: please remove empty lines

File: src/components/utils/ButtonDefault/buttonDefault.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    background-color: $color-white-lm;
+    color: $color-green-lm;
+    
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    flex: 1 0 0;
+  }
+
```
Comment (Meg-Sowka) [inline src/components/utils/ButtonDefault/buttonDefault.scss]: please remove empty lines

File: src/components/utils/ButtonDefault/buttonDefault.scss (hunk 3 above)
Comment (domalewski) [inline src/components/utils/ButtonDefault/buttonDefault.scss]: is this a empty line?

File: src/components/utils/ButtonDefault/buttonDefault.scss (hunk 4 above)
Comment (domalewski) [inline src/components/utils/ButtonDefault/buttonDefault.scss]: omg there's another

File: src/styles/utils/_typography.scss [hunk 5]
```diff
@@ -2,121 +2,121 @@
...
+    font-size: 48px;
+    line-height: 116.667%;
+    letter-spacing: -0.48px;
+  }
+  
+  @include on-desktop {
+    font-size: 48px;
+    line-height: 116.667%;
+    letter-spacing: -0.48px;
+  }
```
Comment (domalewski) [inline src/styles/utils/_typography.scss]: this look's suspicious

**Review by domalewski — CHANGES_REQUESTED** [review]
your good

File: src/styles/utils/_typography.scss (hunk 5 above)
Comment (Matylda-Lewandowska) [reply src/styles/utils/_typography.scss]: what do you mean?

Comment (Matylda-Lewandowska) [reply src/styles/utils/_typography.scss]: ah empty line -.-

**Review by Meg-Sowka — APPROVED** [review]

**Review by domalewski — APPROVED** [review]

**Pull request merged**